src/Controllers/Authentication/login.controller.js

```diff
@@ -3,51 +3,83 @@
 import pool from "../../Database/database";
 import { verify } from "../../middleware/hash-password";
 import { signToken } from "../../middleware/jwt";
+// import jwt from "jsonwebtoken";
+//import Token from "../../models/token";
```

Comment:
remember to remove the unused or commented code block

Reply:
Yeah Thanks Prince, Let me do It.